tests/functional/r/redefined/redefined_loop_name.txt

```diff
@@ -0,0 +1,6 @@
+redefined-loop-name:9:4:9:8::Redefining 'line' from loop (line 8):HIGH
```

Comment:
I think the related test file is missing here?

Reply:
Oh, it got moved under `ext/`, and I forgot to delete this one. Thanks.